### ISSUE TITLE
Consider combining efforts with Configuration Split on Drupal.org

### ISSUE THREAD

**Issue by frederickjh** [previousnext/drush_cmi_tools#11]
Hi!
Looks like have a great project. I was looking for something like this and found your project and [Configuration Split](http://drupal.org/project/config_split) and Drupal.org.

If you have not seen their project I would kindly ask you to consider taking a look at it and see if you can combine your efforts.

Thanks for your time and consideration of this!
Frederick Henderson

**Comment by larowlan** [comment]
Yeah there is merit in joining forces, but I don't want a dev module in production.
As a drush-only command, there is no such issue here.
And I prefer the lightweight nature of the yml files over clicking things together in the UI.
So thanks, but happy as it stands.
